mmpose/datasets/datasets/top_down/topdown_aic_dataset.py

```diff
@@ -0,0 +1,292 @@
+import os
+from collections import OrderedDict, defaultdict
+
+import numpy as np
+from xtcocotools.coco import COCO
+from xtcocotools.cocoeval import COCOeval
+
+from ....core.post_processing import oks_nms, soft_oks_nms
+from ...registry import DATASETS
+from .topdown_coco_dataset import TopDownCocoDataset
+
+
+def _get_mapping_id_name(imgs):
```

Comment:
can we reuse this function? https://github.com/open-mmlab/mmpose/blob/master/mmpose/datasets/datasets/top_down/topdown_ochuman_dataset.py

Reply:
or as commented earlier, 
```
id2name = {id:img['file_name'] for id, img in imgs.items()}
name2id= {img['file_name']:id for id, img in imgs.items()}
```

Reply:
Yes. This func can be reused.